ZConfig/info.py

```diff
@@ -228,6 +220,8 @@ def __init__(self, name, sectiontype, minOccurs, maxOccurs, handler,
         # handler     - handler name called when value(s) must take effect,
         #               or None
         # attribute   - name of the attribute on the SectionValue object
+        assert maxOccurs is not None
+        # because we compare it to 1 which is a Py3 error
```

Comment:
Yeah, I think we should either fully support `maxOccurs` being `None` (maybe that is how you spell "unlimited"?) by adding code to make Python 3 handle this explicitly instead of failing, or we should require it to be not-None everywhere.

I'm sorry that I don't currently have the time to investigate what `maxOccurs` is used for!